CorsixTH/Lua/filesystem.lua

```diff
@@ -75,13 +75,14 @@ function FileSystem:_enumerate()
 end
 
 --! Set the root physical path for this FileSystem.
--- If the path is an ISO then set the provider. If the path is a directory
+-- If the path is a disk image then set the provider. If the path is a directory
 -- then set the pysical_path and populate the files and sub_dirs.
 --
 --!param physical_path (string) a path on the filesystem to either a directory
--- or theme hospital ISO file.
+-- or theme hospital ISO or DMG file.
 function FileSystem:setRoot(physical_path)
-  if physical_path:match"%.[iI][sS][oO]$" or physical_path:match"%.[iI][sS][oO]9660$" then
+  local path = physical_path:lower()
+  if path:match("%.iso$") or path:match("%.iso9660$") or path:match("%.dmg$") then
```

Comment:
Is this Case-Insensitive ?

Reply:
@driverpt There's a tolower just above now. This path is only being used for the match check.